metadata.yaml

```diff
@@ -26,3 +26,4 @@ requires:
   certificates:
     interface: tls-certificates
     limit: 1
+    optional: true
```

Comment:
AFAIK, it changes nothing. At least I saw different variations in our code.
Official [manual](https://juju.is/docs/sdk/metadata-yaml#heading--requires) says: `Informational only`.
Please share if you know what exactly it gives us :-)
Tnx!

Reply:
Yeah! The motivation was to avoid the incessant `unit-zookeeper-0: 15:48:46 WARNING unit.zookeeper/0.juju-log No relation: certificates` logs.

We don't seem to get it on Kafka so was wondering where it came from. Maybe I need to bump the lib.